src/test/java/de/tum/in/www1/artemis/StudentExamIntegrationTest.java

```diff
@@ -590,10 +601,20 @@ public void testSubmitExamOtherUser_forbidden() throws Exception {
         deleteExam1WithInstructor();
     }
 
+    @Test
+    @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
+    public void testgetExamTooEarly_forbidden() throws Exception {
+        prepareStudentExamsForConduction(true);
+
+        database.changeUser("student1");
+
+        var studentExamResponse = request.get("/api/courses/" + course2.getId() + "/exams/" + exam2.getId() + "/student-exams/conduction", HttpStatus.FORBIDDEN, StudentExam.class);
```

Comment:
```suggestion
        request.get("/api/courses/" + course2.getId() + "/exams/" + exam2.getId() + "/student-exams/conduction", HttpStatus.FORBIDDEN, StudentExam.class);
```
Test passes or fails by the return value of the REST call, no variable needed